src/app/clusters/descriptor/descriptor.cpp

```diff
@@ -54,6 +55,25 @@ class DescriptorAttrAccess : public AttributeAccessInterface
 
 constexpr uint16_t DescriptorAttrAccess::ClusterRevision;
 
+CHIP_ERROR DescriptorAttrAccess::ReadTagAttribute(EndpointId endpoint, AttributeValueEncoder & aEncoder)
+{
+    CHIP_ERROR err = aEncoder.EncodeList([&endpoint](const auto & encoder) -> CHIP_ERROR {
```

Comment:
Since the taglist is dependent on the feature bit, this probably needs to check the feature bit before getting the attribute.

Reply:
We can't use `Descriptor::Attributes::featureMap::Get(endpoint, &featureMap);` to get cluster-revision in app. So it should make sure to set in zap